mui/scripts/linkComponents.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+BLUE='\033[0;34m'
+BBLUE='\033[1;34m' #BOLD
+PURPLE='\033[0;35m'
+RED='\033[0;31m'
+BRED='\033[1;31m' #BOLD
+GREEN='\033[0;32m'
+BGREEN='\033[1;32m' #BOLD
+GRAY='\033[1;30m'
+NC='\033[0m' # No Color
+
+echo -e "${BLUE}Building components...${NC}"
+yarn build
+
+echo -en "${BLUE}Creating new folder in node_modules...${NC}"
+rm -rf "./demos/showcase/node_modules/@pxblue/icons-mui"
+mkdir -p "./demos/showcase/node_modules/@pxblue/icons-mui"
+# rm -rf "./demos/storybook/node_modules/@pxblue/icons-mui"
+# mkdir -p "./demos/storybook/node_modules/@pxblue/icons-mui"
+echo -e "${GREEN}Done${NC}"
+
+echo -en "${BLUE}Copying build output into node_modules...${NC}";
+# cp -r ./icons/package.json ./demos/showcase/node_modules/@pxblue/icons-mui/package.json
+cp -r ./icons/. ./demos/showcase/node_modules/@pxblue/icons-mui
+# cp -r ./components/package.json ./demos/storybook/node_modules/@pxblue/icons-mui/package.json
+# cp -r ./dist/. ./demos/storybook/node_modules/@pxblue/icons-mui
```

Comment:
I think the first three lines here are redundant (and pulling from the wrong folder). The last linecopies everything from the dist folder, which should be everything we need.

Reply:
I'm not sure how, when, if this linking script is used anymore but this should be fixed now.

Reply:
It's used to link the components up to the demo. I think this file was probably originally copied from the component library which has multiple demo projects.